pepper-apis/src/main/java/org/broadinstitute/ddp/analytics/GoogleAnalyticsMetricsTracker.java

```diff
@@ -0,0 +1,121 @@
+import org.broadinstitute.ddp.db.dao.StudyDao;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.model.study.StudySettings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GoogleAnalyticsMetricsTracker {
+    private static final Logger LOG = LoggerFactory.getLogger(GoogleAnalyticsMetricsTracker.class);
+    private static final Integer DEFAULT_BATCH_SIZE = 2;
+    public static Map<String, GoogleAnalytics> studyAnalyticsTrackers = new HashMap<>();
+    private static Set<String> noAnalyticsTokenStudies = new HashSet<>(); //studyGuid with NO analytics token
+
+    private static GoogleAnalytics getMetricTracker(String studyGuid) {
+        if (!studyAnalyticsTrackers.containsKey(studyGuid) && !noAnalyticsTokenStudies.contains(studyGuid)) {
+            initStudyMetricTracker(studyGuid);
+        }
+        return studyAnalyticsTrackers.get(studyGuid);
+    }
+
+    private static synchronized void initStudyMetricTracker(String studyGuid) {
+        if (!studyAnalyticsTrackers.containsKey(studyGuid)) {
+            StudySettings studySettings = getStudySettingByStudyGuid(studyGuid);
+            String studyTrackingId = studySettings.getAnalyticsToken();
+            if (StringUtils.isEmpty(studyTrackingId)) {
+                LOG.error("NO analytics token found for study : {} . skipping sending analytics. ", studyGuid);
+                noAnalyticsTokenStudies.add(studyGuid);
+                return;
+            }
+
+            GoogleAnalytics metricTracker = GoogleAnalytics.builder()
+                    .withConfig(new GoogleAnalyticsConfig().setBatchingEnabled(true).setBatchSize(DEFAULT_BATCH_SIZE))
+                    .withTrackingId(studyTrackingId)
+                    .build();
+            studyAnalyticsTrackers.put(studyGuid, metricTracker);
+            LOG.info("Initialized GA Metrics Tracker for study GUID: {} ", studyGuid);
+        }
+    }
+
+    public static void sendEventMetrics(String studyGuid, EventHit eventHit) {
+        //if (noAnalyticsTokenStudies.contains(studyGuid) || getStudySettingByStudyGuid(studyGuid).isAnalyticsEnabled()) {
+        if (noAnalyticsTokenStudies.contains(studyGuid)) {
+            return;
+        }
+
+        GoogleAnalytics metricTracker = getMetricTracker(studyGuid);
+        if (metricTracker != null) {
+            metricTracker.event().eventCategory(eventHit.eventCategory())
+                    .eventAction(eventHit.eventAction())
+                    .eventLabel(eventHit.eventLabel())
+                    .eventValue(eventHit.eventValue())
+                    .sendAsync();
+        }
+    }
+
+    public static StudySettings getStudySettingByStudyGuid(String studyGuid) {
+        //todo: revisit after Redis caching to use cached Study, StudySettings and get rid of StudySettingsStore
+        return StudySettingsStore.getInstance().getStudySettings(studyGuid);
+    }
+
+
+    private static class StudySettingsStore {
```

Comment:
Caching of StudySettings. Once Redis caching PR is merged, this can be removed and refactored to use Redis Study, StudySettings caching.